jsonpickle/unpickler.py

```diff
@@ -624,7 +624,13 @@ def _restore_from_dict(self, obj, instance, ignorereserved=True):
                     continue
             else:
                 if not k.startswith('__'):
-                    setattr(instance, k, value)
+                    try:
+                        setattr(instance, k, value)
+                    except KeyError:
+                        # certain numpy objects require us to prepend a _ to the var
+                        # this should go in the np handler but I think this could be
+                        # useful for other code
+                        setattr(instance, f"_{k}", value)
```

Comment:
Thanks for including this comment. A little hacky, but hey, if it works it's all good.